From 0efa918cf6ec1d70c527241317783eefab6b0b2d Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Tue, 24 Jul 2018 17:14:09 +0800
Subject: [PATCH 2/3] driver: usb: ehci: remove redefined
 "ehci_has_fsl_susp_errata"

To fix:
drivers/usb/host/ehci.h:726:0: warning: "ehci_has_fsl_susp_errata" redefined
 #define ehci_has_fsl_susp_errata(e) ((e)->has_fsl_susp_errata)

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/usb/host/ehci.h |    7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/usb/host/ehci.h b/drivers/usb/host/ehci.h
index cc881cb..67f3cc2 100644
--- a/drivers/usb/host/ehci.h
+++ b/drivers/usb/host/ehci.h
@@ -719,13 +719,6 @@ struct ehci_tt {
 #endif
 
 /*
- * Some Freescale/NXP processors have an erratum (USB A-005697)
- * in which we need to wait for 10ms for bus to enter suspend mode
- * after setting SUSP bit.
- */
-#define ehci_has_fsl_susp_errata(e)	((e)->has_fsl_susp_errata)
-
-/*
  * While most USB host controllers implement their registers in
  * little-endian format, a minority (celleb companion chip) implement
  * them in big endian format.
-- 
1.7.9.5

